### PR TITLE
add missing types & docs

### LINKED ISSUE #305: Documentation
Add `queue.totalTime` and the `MusicStarting` event to the docs.

### LINKED ISSUE #309: Add queue.currentStreamTime to the docs? And add queue.totalTime to the docs.
`queue.currentStreamTime` seems very useful sometimes, and it would be nice to see this in the docs. Also, `queue.totalTime` is missing from the docs.

### PR DESCRIPTION
**Pull request by Dev-CasperTheGhost**
## This PR includes

- Added missing documentation for few Queue getters
- Added missing TypeScript types for `Queue#totalTime` and `PlayerError`

closes #305 and closes #309

### CODE REVIEW

File: src/Queue.js [hunk 1]
```diff
@@ -90,18 +90,34 @@ class Queue extends EventEmitter {
...
+     * The current playing track
+     * @type {Track}
+     */
     get playing () {
         return this.tracks[0]
     }
 
+    /**
+     * The calculated volume of the queue
+     * @type {Number}
```
Comment (Androz2091) [inline src/Queue.js:103]:
```suggestion
     * @type {number}
```

File: src/Queue.js [hunk 2]
```diff
@@ -90,18 +90,34 @@ class Queue extends EventEmitter {
...
+     */
     get calculatedVolume () {
         return this.filters.bassboost ? this.volume + 50 : this.volume
     }
 
+    /**
+     * Returns the total time of the queue in milliseconds
+     * @type {Number}
```
Comment (Androz2091) [inline src/Queue.js:111]:
```suggestion
     * @type {number}
```

File: src/Queue.js [hunk 3]
```diff
@@ -90,18 +90,34 @@ class Queue extends EventEmitter {
...
+     */
     get totalTime () {
         return this.tracks.length > 0 ? this.tracks.map((t) => t.durationMS).reduce((p, c) => p + c) : 0
     }
 
+    /**
+     * The current stream time
+     * @type {Number}
```
Comment (Androz2091) [inline src/Queue.js:119]:
```suggestion
     * @type {number}
```

**Review by Androz2091 — CHANGES_REQUESTED** [review]
Thank you 👍 
here are some changes, built in types are lowercase in all the rest of the code

**Comment by Dev-CasperTheGhost** [comment]
Updated the types to lowercase :)

**Review by Androz2091 — APPROVED** [review]
Thank you 👍

**Pull request merged**